internal/functions/efgs/api/structs.go

```diff
@@ -96,15 +96,16 @@ type DiagnosisKeyWrapper struct {
 	tableName                  struct{}  `pg:"diagnosis_keys,alias:dk"`
 	ID                         int32     `pg:",pk" json:"id"`
 	CreatedAt                  time.Time `pg:"default:now()" json:"created_at"`
-	KeyData                    []byte    `json:"keyData,omitempty"`
```

Comment:
I know it will be inconvenient at few places but could you change this column to string (base64)? It would make testing MUCH easier..